test/ttmlir/Conversion/StableHLOToTTIR/transpose_op.mlir

```diff
@@ -0,0 +1,10 @@
+// REQUIRES: stablehlo
```

Comment:
Please move this test to `test/ttmlir/Conversion/StableHLOToTTIR/unary/transpose_op.mlir`

Reply:
Done

Reply:
I also changed the name of the transpose test that was originally there.